src/features/message-center/components/templates/preview-title-body-tags-button/PreviewTitleBodyTagsButton.js

```diff
@@ -1,12 +1,13 @@
 // @flow
 import * as React from 'react';
 import type { Token } from '../../../../../common/types/core';
-import MessagePreviewContent from '../common/MessagePreviewContent';
+import MessagePreviewContent from '../../../../../components/message-preview-content/MessagePreviewContent';
 import MessageTextContent from '../common/MessageTextContent';
 import MessageTags from '../common/MessageTags';
 import MessageFooter from '../common/MessageFooter';
 import BottomContentWrapper from '../common/BottomContentWrapper';
-import type { ContentPreviewProps, PreviewTitleBodyTagsButtonTemplateParams } from '../../../types';
+import type { PreviewTitleBodyTagsButtonTemplateParams } from '../../../types';
+import type { ContentPreviewProps } from '../../../../../common/types/contentPreview';
```

Comment:
Rather than needing to import this everywhere, can we import it in `MessagePreviewContent` and re-export it as `MessagePreviewContentProps`?

Reply:
Done, but I'm not sure how it improves the import, could you elaborate on that a little bit more?